test/context.test.js

```diff
@@ -1,8 +1,8 @@
 import React from "react"
 import * as mobx from "mobx"
 import { Provider, observer, inject } from "../src"
-import { withConsole } from "./"
 import TestRenderer from "react-test-renderer"
+import { withConsole } from "."
```

Comment:
Um, can we actually have it in a `withConsole.ts` file? These imports from the index are strange :)

Reply:
I will create `test/utils/withConsole.js` module, if you do not mind.

Reply:
Sure, that works as well :)

Reply:
It's done.

Reply:
Almost...check tests failure :)

Reply:
I'm sorry. I forgot to add some changes to Git 🤦🏻‍♂️